README.md

```diff
@@ -77,65 +111,63 @@ is described in 1 step and a step later mentions the same filename, they both re
       * [breakends.vcf.gz](https://github.com/molgenis/capice/blob/master/tests/resources/breakends.vcf.gz)
       * [edge_cases.vcf.gz](https://github.com/molgenis/capice/blob/master/tests/resources/edge_cases.vcf.gz)
       * [symbolic_alleles.vcf.gz](https://github.com/molgenis/capice/blob/master/tests/resources/symbolic_alleles.vcf.gz)
-   6. Update [utility_scripts/slurm_run_vep_step1.sh](utility_scripts/slurm_run_vep_step1.sh):
-      * Ensure all paths are correct.
-      * Ensure the VEP command is equal to that of the CAPICE readme, except: 
-        * `--per_gene` is included.
-        * `--buffer_size` is added (f.e. 500) to reduce memory usage if needed.
-   7. Run the slurm job:
-      ```shell
-      sbatch slurm_run_step1.sh
-      ```
-      This will generate the following files:
+   6. Annotate all downloaded files with VEP using the supplied [slurm_run_vep.sh](utility_scripts/slurm_run_vep.sh):
+       * Supply a smaller `--time` argument to slurm (processing the files should take a maximum of 20 minutes each)
+       * Ensure `-g` is supplied.
+       * To reduce potential error, a for loop should be used to mark each file (this is assuming you have changed directory into the single directory):
+         * ```bash
+           for file in *.vcf.gz; do sbatch --time=00:20:00 slurm_run_vep.sh -p </path/to/vip_install_directory> -i "${file}" -g -o "${file%.vcf.gz}_vep.vcf.gz"; done
+           ```
+       * Once all files have been processed, rename the following files:
+       * ```bash
+         mv ./train_input_raw_vep.vcf.gz ./train_input_annotated.vcf.gz
+         mv ./predict_input_raw_vep.vcf.gz ./predict_input.vcf.gz
+         ```
+       You should now have the following files:
       - train_input_annotated.vcf.gz
       - predict_input.vcf.gz
       - breakends_vep.vcf.gz
       - edge_cases_vep.vcf.gz
       - symbolic_alleles_vep.vcf.gz
-   8. Load BCF tools and run
-      [CAPICE conversion tool](https://github.com/molgenis/capice/blob/master/scripts/convert_vep_vcf_to_tsv_capice.sh)
+   7. Run the [CAPICE conversion tool](https://github.com/molgenis/capice/blob/master/scripts/convert_vep_vcf_to_tsv_capice.sh)
       on the train input. (note: use `-t` when using the conversion tool)
       ```shell
-      module load BCFtools/1.11-GCCcore-7.3.0
```

Comment:
* Why is this removed?
* Might need to be updated to current most recent on cluster (`BCFtools/1.16-GCCcore-11.3.0`)

Reply:
It was assumed that the [apptainer CAPICE PR](https://github.com/molgenis/capice/pull/156) is approved and merged by the time this PR is merged.